index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title> Maddie's Portfolio </title>
+</head>
+<body>
+    <link rel="stylesheet", href="style/portfolio.css">
+    <main>
+      <header>
+        <h1> Maddie's Portfolio </h1>
+      </header>
+      <aside>
+        <p> Aside </p>
+      </aside>
+        <section id="projects">
+          <article class="template", data-category="cat">
+            <header>
+              <h3> Project Title</h3>
+            </header>
+          <section class="projectDescription"> </section>
+        </article>
+        </section>
+      </main>
+      <footer>
+        <p> CONTACT INFORMATION: <a href="https://github.com/jdar22169"> GitHub </a> and
+          <a href="https://www.linkedin.com/in/madelinemow"> LinkedIn </a> </p>
+      </footer>
```

Comment:
Watch your indentations. You have lines that are indented 4 spaces and some 2 spaces--they should be consistent. Also, make sure the closing tags line up with their corresponding tags.